src/main/java/me/creighton/encodedid/profiles/OTPEncoder.java

```diff
@@ -61,9 +73,9 @@ public long getLargestOtpId() {
   @Override
   public String encode () {
     long min = this.minOtpId;
-    long max = (this.maxOtpId < Long.MAX_VALUE) ? this.maxOtpId : this.maxOtpId;
+    long max = this.maxOtpId;
 
-    long id = getRandomLong(max, min);  //
+    long id = getRandomLong(min, max);  //
```

Comment:
I like the change in parameter order - max is typically after the min.


Minor feedback - empty comment.
```suggestion
    long id = getRandomLong(min, max);
```